.github/workflows/linux.yml

```diff
@@ -43,15 +43,15 @@ jobs:
     steps:
       - uses: actions/checkout@v2.0.0
       - name: Cache Build
-        uses: actions/cache@v1.1.0
+        uses: actions/cache@v2
         with:
           path: build
-          key: ${{ matrix.os }}-cmake-${{ matrix.EVENT_MATRIX }}-v2
+          key: ${{ matrix.os }}-cmake-${{ matrix.EVENT_MATRIX }}-v3
```

Comment:
Bumping cache key is not a long-term solution, can you take a look why it is required at first place? (remove the cache on failure and retry should work, that you added for cmake, but I guess that there is something left)

But can be done separately

Reply:
I observed the recent CI failures caused by caching, and found that an important reason is that github action updates the virtual environment version (once a week, some software versions are updated, such as cmake, vcpkg, etc.). 
In fact, clearing the build cache can solve it Part of the problem, but the wrong cache will not be updated.

In addition, the existing Dependes cache may also cause CI failures. Because the key that our Dependes depends on already exists, the new dependency mbedtls will never be installed.